packages/vue-components/src/__tests__/__snapshots__/Modal.spec.js.snap

```diff
@@ -3,22 +3,22 @@
 exports[`Modal can be centered 1`] = `
 <div
   class="vfm vfm--inset vfm--fixed"
-  data-v-47686880=""
```

Comment:
Any reason why this is updated? Due to caching?

Always just want to make sure: is this a one-time change? Or will running updatetest modify this again?

Reply:
This happens whenever npm i is run.
I think this occurs when one of the package is updated (I did not pinpoint exactly which).

This is probably not a one-time change. Next time someone updates that specific package again this will occur ~. But given how rarely we update we run npm i, this should occur rarely.

Reply:
what about running npm run setup? Does it make this test file update? If it's due to npm i, is this an unrelated change to this lerna update or because of it?

If it's unintentional, we should find out the root cause for it.

Reply:
It is a unique attribute generated by Vue.js at runtime. The identifier is used to ensure that the component's styles only apply to elements within the component and not elsewhere on the page.

It is expected behavior for vue. (prob due to seed changing in vue)

npm run setup => does not cause this as it runs npm ci.

Reply:
That's interesting. I would assume (and accept) if this change only occurs during npm run updatetest, but you are saying it occurs even when invoking npm i? Why so? During npm i, I'm not running Vue.js at runtime?

Reply:
The reason why the value of data-v- changes when libraries are updated is because the unique identifier is generated by Vue.js at runtime, and the algorithm for generating this identifier may change between different versions of Vue.js or other libraries that the application depends on.

Some libraries updated => unique value changes => testcases no longer pass => npm updattest (which reflects the value being changed.

Reply:
I see, thanks for explaining. In that case, let's find out which library causes it.

Reply:
Thanks @tlylt for the pointers.
So to sum up.
1. Running npm i directly may  result in data-v changing.
2. But npm i can be avoided 99% of the time, for this case we needed to init the npm workplaces thus we need to run npm I. Any other occation we can npm i <specific-package>@<specific version> to avoid updating unrelated dependencies.

All in all, data-v will only be updated rarely unless vue, css or webpack related packages are updated.